src/test/resources/org/jabref/logic/importer/fileformat/MsBibImporterTestTranslator.bib

```diff
@@ -0,0 +1,15 @@
+% Encoding: UTF-8
+
+@Misc{2016,
+  title          = {MeinArtikel},
```

Comment:
This doesn't look like the bibtex version of the xml file below.

Reply:
Thanks for noticing. But shouldn't travis normally have noticed this?